content/blog/bonsai-0-1-0/index.Rmd

```diff
@@ -0,0 +1,193 @@
+---
+output: hugodown::hugo_document
+
+slug: bonsai-0-1-0
+title: bonsai 0.1.0
+date: 2022-06-30
+author: Simon Couch
+description: >
+    A new parsnip extension package for tree-based models is now on CRAN.
+
+photo:
+  url: https://unsplash.com/photos/-OBffuUekfQ
+  author: 五玄土
+
+categories: [package] 
+tags: [tidymodels, parsnip, bonsai]
+---
+
+We're super stoked to announce the first release of the [bonsai](https://bonsai.tidymodels.org/) package on CRAN! bonsai is a [parsnip](https://parsnip.tidymodels.org/) extension package for tree-based models.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("bonsai")
+```
+
+Without extension packages, the parsnip package already supports fitting decision trees, random forests, and boosted trees. The bonsai package introduces support for two additional engines that implement variants of these algorithms:
+
+* [partykit](https://CRAN.R-project.org/package=partykit): conditional inference trees via `decision_tree()` and conditional random forests via `rand_forest()`
+* [LightGBM](https://CRAN.R-project.org/package=lightgbm): optimized gradient boosted trees via `boost_tree()`
+
+To demonstrate how to use the package, we'll fit a few tree-based models and explore their output. First, loading bonsai as well as the rest of the tidymodels core packages:
+
+```{r}
+library(bonsai)
+
+library(tidymodels)
```

Comment:
Re-render with CRAN model data please

Reply:
ok, you use it later. Maybe mention that you are using that version for a reason and give commented out instructions on installing.